README.md

```diff
@@ -17,8 +17,10 @@ Pass your Checkout URL when initialising a new TrustlyWKWebView instance. The Ch
 
 Example usage:
 ```swift
-let trustlyWebView = TrustlyWKWebView(checkoutUrl: trustlyCheckoutURLString, frame: self.view.frame)
-self.view = trustlyWebView 
+let webViewController = UIViewController.init(nibName: nil, bundle: nil)
+let mainView = TrustlyWKWebView(checkoutUrl: checkoutUrl.absoluteString, frame: self.view.bounds)
```

Comment:
Maybe the `checkout.absolutString` can be replace with a placeholder like so `[checkout Url]` or something that's a bit more generic?

Reply:
Done